bin/lesson_check.py

```diff
@@ -79,12 +78,16 @@
     'source',
     'language-bash',
     'html',
+    'language-c',
+    'language-cmake',
+    'language-cpp',
     'language-make',
     'language-matlab',
     'language-python',
     'language-r',
     'language-shell',
-    'language-sql'
+    'language-sql',
+    'warning'
```

Comment:
I guess you could add `language-yaml` here for a local fix of the issue mentioned above?

Reply:
I added `--permissive` to the Makefile, so these warnings are no longer fatal. This is ready to go, if you want to approve it :-)

Reply:
And as we have a lot of self-inflicted FIXMEs we have to have `--permissive`

Reply:
That works for now, I guess.